### PR TITLE
schedule dencun on holesky

### PR DESCRIPTION
**Pull request by g11tech**
see:
 - https://github.com/ethereum/EIPs/pull/8051

### CODE REVIEW

**Review by philknows — APPROVED** [review]

**Comment by pk910** [comment]
I think the cancun timestamp should be added to the EL genesis (genesis.json / besu.json / chainspec.json) too?
```
"cancunTime": 1701347280
```

**Comment by g11tech** [comment]
> I think the cancun timestamp should be added to the EL genesis (genesis.json / besu.json / chainspec.json) too?
> ```
> "cancunTime": 1701347280
> ```

Ahh yes this has full custom config, makes sense will update 👍

**Review by pk910 — APPROVED** [review]

**Review by parithosh — APPROVED** [review]

**Review by barnabasbusa — APPROVED** [review]

**Review by Geewhiz999 — APPROVED** [review]

File: custom_config_data/genesis.json
Comment (Geewhiz999) [inline custom_config_data/genesis.json:1]: custom_config_data/genesis.json

Comment (Geewhiz999) [reply custom_config_data/genesis.json:1]: Thanks

Comment (Geewhiz999) [reply custom_config_data/genesis.json:1]: > custom_config_data/genesis.json

**Review by tersec — APPROVED** [review]

**Comment by realbigsean** [comment]
this config's missing `MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 8`

**Comment by realbigsean** [comment]
made a PR for it here https://github.com/eth-clients/holesky/pull/95

**Pull request merged**